airflow/www/api/experimental/endpoints.py

```diff
@@ -234,3 +235,20 @@ def delete_pool(name):
         return response
     else:
         return jsonify(pool.to_json())
+
+
+@csrf.exempt
+@api_experimental.route('/dags/<string:dag_id>/remove_dag', methods=['POST'])
```

Comment:
For this endpoint, I suggest to user 'DELETE' method and also remove `/remove_dag`. The route will become
`@api_experimental.route('/dags/<string:dag_id>', methods=['DELETE'])`